content/_index.md

```diff
@@ -48,6 +48,12 @@ To engage with specific content please file issues and / or PRs directly to the
 
 To stay up to date on the group's activities there's also a Twitter account you can follow: [uapi-group](https://twitter.com/uapi_group).
 
+## Governance
+
+The UAPI-group and its specifications are governed in similar ways to other free-software projects.
+Every maintainer has the right to merge to all the repositories, and might do so under the assumption that they hold the expertise needed to do so.
```

Comment:
maybe add "(for non-trivial contributions it is expected that maintainers shall wait for review by at least one other maintainer before merging their own PRs)" or so.

Reply:
Now a third point reads: "Except for trivial changes, an approval from at least one maintainer who is not the author is required"